### ISSUE TITLE
Namespace

### ISSUE THREAD

**Issue by Rapafull** [mono0926/barcode_scan2#78]
A problem occurred configuring project ':barcode_scan2'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file.

**Comment by ArocaAlejandro** [comment]
Downgrade your build to gradle 7.

Only need to add the namespace for the module in the package for working with the newest requisites from gradle 8.x. Can do it manually until they fix it entering in the package and adding the namespace="de.mintware.barcode_scan"

**Comment by mericgerceker** [comment]
any update on the?

**Comment by ArocaAlejandro** [comment]
Is the same. If anyone tries to update internally and build for do a PR, the new Gradle for Flutter 3.16+ break a lot of things, 

In a few days, for low time day, i've tried to do this, understand firstly the code and change the minor things deprecated. But I need more time for see why it breaks and doesn't compile. But if we change this file this compiles and doesn't break anything. But i don't know if the tests pass doing this or if the app crashes using the package with this config.

**Comment by pst9354** [comment]
Anyone being able to do a PR to solve this issue?

**Comment by robinbonnes** [comment]
I've created a [PR](https://github.com/mono0926/barcode_scan2/pull/87), you can use my fork untill this is merged by changing your `pubspec.yaml` to:

```
dependencies:
  barcode_scan2:
    git:
      url: https://github.com/robinbonnes/barcode_scan2
```

**Comment by mono0926** [comment]
Merged #87 and released v4.3.2 🐶
